ui/src/message_popup/summary_card.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import Divider from 'material-ui/Divider';
+import VelocityTransitionGroup from 'velocity-react/velocity-transition-group';
+
+
+export default React.createClass({
+  displayName: 'SummaryCard',
+  
+  getInitialState(){
+    return ({
```

Comment:
Does this need to be `state`, or can it just be a prop that's passed in?  It doesn't seem to be changing anywhere in this component.

Also, from the name `timer` it's hard to know what it is, here it looks like maybe it's `elapsedSeconds`?